### PR TITLE
fix: swap style updates

### PR DESCRIPTION
**Pull request by kayleegeorge**
swap design updates implemented with feature flags

without feature flag on (old):
Screen Shot 2022-08-03 at 2 47 28 PM

with feature flag on (new):
Screen Shot 2022-08-03 at 2 51 41 PM

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -71,7 +71,7 @@
         -webkit-font-smoothing: antialiased;
         -moz-osx-font-smoothing: grayscale;
         -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
-        font-feature-settings: 'ss01' on, 'ss02' on, 'cv01' on, 'cv03' on;
+        font-feature-settings: 'ss01' on, 'ss02' off, 'cv01' on, 'cv03' on;
```
Comment (vm) [inline public/index.html:74]: what is this?

Comment (kayleegeorge) [reply public/index.html:74]: font turning off the slash in the 0

**Comment by vm** [comment]
can we close this?

**Comment by kayleegeorge** [comment]
moved here: https://github.com/Uniswap/interface/pull/4294

**Pull request closed without merging**